sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -135,14 +141,14 @@ void WinSocketSetBuffSize(curl_socket_t socket)
     // Specifies the total per-socket buffer space reserved for sends.
     // https://docs.microsoft.com/windows/win32/api/winsock/nf-winsock-setsockopt
     auto result = setsockopt(socket, SOL_SOCKET, SO_SNDBUF, (const char*)&ideal, sizeof(ideal));
-
-    if (Log::ShouldWrite(Logger::Level::Verbose))
-    {
-      Log::Write(
-          Logger::Level::Verbose,
-          LogMsgPrefix + "Windows - calling setsockopt after uploading chunk. ideal = "
-              + std::to_string(ideal) + " result = " + std::to_string(result));
-    }
+    result;
```

Comment:
what does `result;` does?

You can remove this log, no need to keep it commented.

Reply:
`result;` makes the C++ compiler happy.

Reply:
We typically do `(void)result;` to suppress the un-used variable.

I'd suggest to follow the same syntax